src/platform.ts

```diff
@@ -108,15 +108,15 @@ export class KumoHomebridgePlatform implements DynamicPlatformPlugin {
         const overrideAddress = this.optionGetOverrideAddress(device);
         if (overrideAddress !== null) {
           this.log.info('Override address found for device - using IP %s instead of %s for direct access', overrideAddress, existingAccessory.context.zoneTable.address);
-          existingAccessory.context.overrideAddress = overrideAddress;
+          device.overrideAddress = overrideAddress;
```

Comment:
Need to set it on the device record itself, not on the context (outside of the API)